src/components/Stepper/Stepper.js

```diff
@@ -224,6 +249,7 @@ function Stepper({ steps, onComplete, ...props }) {
           {layout === 'expanded' && renderSteps()}
         </ul>
       </div>
+      {renderNextActions()}
```

Comment:
if the answer for the previous question is that you thought about it to render anything i think that might make sense to have the if inside the html as we do always instead of having the function call

Reply:
I preferred to call the function and return null if conditions are not fulfilled so we don't have too much logic at the render level

Reply:
i got your point for when we have a really complex logic, but in this case does not apply, we barely have html rendered on functions in our codbase and the good thing about having all in the html is that you don't need to take out your view to the logic part of the component to follow the html tree. so in this case i would stick as we usually do

Reply:
Ok that's fair!